bskyweb/templates/post.html

```diff
@@ -33,6 +37,15 @@
   {# Don't use avatar image in cards; usually looks bad #}
   <meta name="twitter:card" content="summary">
   {% endif %}
+  {%- if videoUrl %}
+  <meta property="og:video:type" content="video/mp4">
+  <meta property="og:video:height" content="{{ height }}">
+  <meta property="og:video:width" content="{{ width }}">
+  <meta property="og:image:height" content="{{ height }}">
+  <meta property="og:image:width" content="{{ width }}">
+  <meta property="og:video:secure_url" content="{{ videoUrl }}">
```

Comment:
I see `secure_url` in the OG spec page, but it feels legacy/remnant unless some systems actually provide it or there is clear authoritative guidance that it is still required. We live in an `https://` world!